index.ts

```diff
@@ -18,6 +18,13 @@ declare namespace ethereum {
 /** Host IPFS interface */
 export declare namespace ipfs {
   function cat(hash: string): Bytes | null
+  function map(hash: String, callback: String, flags: String[]): void
```

Comment:
Lowercase `string` please, there are subtle differences between them and I remember `string` to be the better choice — plus we use it everywhere.

Reply:
Fixed